### PR TITLE
Release/2023 05 02

### PR DESCRIPTION
**Pull request by ludtkemorgan**
Pulls the following changes over from core:

* 28f28708d - fix: partners highlight field on backend error (#3448)
* 95d181bdf - feat: adding knownError flag 
* 8dfe15d1a - fix: missing state data (#3450)
* 813a53d4d - feat: 3291/listing export take 2 (#3424)
* f6818961d - fix: update ui-c to latest version (#3420) 
* fe7664f56 - fix: remove check in test not applicable for hba 
* d25db2f3a - fix: now removes criteria file if a url is input (#3421) 
* 078f06315 - fix: downgrade version of nest axios (#3419) 
* 2a0b66445 - fix: add translation for 64 characters error (#3423)
* 966426dc0 - (main) updates proxy to support access control allow list (#3407)
* 1e164ba93 - fix: Searching on the applications table causes the page to crash (#3408)
* b58f8cf3b - fix: updates around cookies (#3405) 
* 5f9fd43de - chore(deps): bump cookiejar from 2.1.2 to 2.1.4 (#3295)
* b96e5e1b8 - feat: removing sensative info from leasing agent (#3409)
* c964f34af - fix: add three new fields to base view (#3406) 
* 80749a5dd - fix: add startDate to open house submit event (#3399)
* d2dd78a17 - refactor: uptake seeds FormErrorMessage (#3369) 
* 9cf0d06cb - refactor: add cloudinary fxn to partners (#3393)
* 0aee30f7b - fix: react type errors (#3382) 
* 77df46250 - fix: resolves issues around markedAsDuplicate (#3373) 
* 5eedba1dc - feat: changing auth over to cookies (#3357) 
* 453e0ef65 - feat: upgrade nextjs to 13 (#3375) 
* 7d89d5c55 - feat: upgrade react to 18 (#3360) 
* 377e4e3d6 - fix: uptake latest uic Modal, ActionBlock, FormCard breaking changes (#3358)

### CODE REVIEW

**Comment by ColinBuyck** [comment]
Hmmm 🍍 anytime I run this locally and try to apply to a listing, I get a Unhandled Runtime Error and see the following in the terminal (occurred for multiple listings)
Screenshot 2023-05-08 at 10 57 56 AM

**Comment by ludtkemorgan** [comment]
@ColinBuyck I am unable to reproduce that. At what point do you get the error? Is it right when you click the confirm button? It looks like it fails on an options call which is weird

**Review by ColinBuyck — APPROVED** [review]
LGTM 🪕

**Comment by emilyjablonski** [comment]
Is this ready for another set of eyes after the bash?

**Comment by ludtkemorgan** [comment]
@emilyjablonski, yes this can be reviewed. This bug is preventing us from releasing https://github.com/bloom-housing/bloom/issues/3430. But the code can be looked at and I would like to get this merged, we just can't release

File: sites/partners/src/components/listings/PaperListingForm/sections/ListingIntro.tsx [hunk 1]
```diff
@@ -60,7 +61,12 @@ const ListingIntro = (props: ListingIntroProps) => {
             keyPrefix={"jurisdictions"}
             options={jurisdictionOptions}
             inputProps={{
-              onChange: () => clearErrors("jurisdiction"),
+              onChange: () => {
+                console.log("jurisdiction change")
```
Comment (YazeedLoonat) [inline sites/partners/src/components/listings/PaperListingForm/sections/ListingIntro.tsx:65]: may want to pull this console log out

**Review by YazeedLoonat — APPROVED** [review]
LGTM
just a console log that snuck in

**Review by YazeedLoonat — APPROVED** [review]
LGTM
just a console log that snuck in

**Pull request merged**